iwyu_ast_util.cc

```diff
@@ -1245,6 +1255,14 @@ const NamedDecl* TypeToDeclAsWritten(const Type* type) {
   }
 }
 
+const NamedDecl* TypeToDeclAsWritten(const Type* type) {
+  return TypeToDeclImpl(type, true);
```

Comment:
You could add a `/*as_written=*/true` here to make it clearer what the flag means. Cheapest form of named arguments.

Reply:
Done.